test/test_cases/TEST_ID_10_SECRETS_DESTINATION_not_exist.sh

```diff
@@ -12,4 +12,4 @@ deploy_test_env
 pod_name=$(cli_get_pods_test_env | awk '{print $1}')
 
 echo "Expecting secrets provider to fail with error 'CSPFK004E Environment variable 'SECRETS_DESTINATION' must be provided'"
-wait_for_it 600 "$cli logs $pod_name -c cyberark-secrets-provider | grep 'CSPFK004E'"
+$cli "logs $pod_name -c cyberark-secrets-provider | grep CSPFK004E"
```

Comment:
why was this removed?

Reply:
i suggest you to go to utils function and you will see the change. all cli command will run wait_for_it as part of the command

Reply:
saw it :) Asked some questions there.